### ISSUE TITLE
Art mistake in "Dodge the creeps" first 2d game tutorial

### ISSUE THREAD

**Issue by chuckdries** [godotengine/godot-docs#5936]
**Your Godot version:**
v3.4.4.stable.mono.official [419e713a2]

**Issue description:**
Hello! I'm following the "dodge the creeps" getting started tutorial, and I'm on the "coding the player" section. I've got to the point where the script switches which animation is playing based on the direction, and mirrors the art vertically or horizontally depending on direction as well. The problem is that the tutorial states that the player should point all 4 cardinal directions while moving, but my player never rotates left or right, only up or down. I believe the issue is that the provided art assets `playerGrey_walk1.png` and `playerGrey_walk2.png` should be rotated 90 degrees, as the file names and the code in the tutorial suggest that the walk animation should show the player moving right and left, but the art assets point up and are identical to the `..._up` assets provided.

**URL to the documentation page:**
https://docs.godotengine.org/en/stable/getting_started/first_2d_game/03.coding_the_player.html#choosing-animations

**Comment by ego-edesse** [comment]
Not sure to get your point, tell me if I misunderstood !
The player sprite actually rotates both vertically and horizontally. Look at the code

```
if velocity.x != 0:
    $AnimatedSprite.animation = "walk"
    $AnimatedSprite.flip_v = false
    # See the note below about boolean assignment.
    $AnimatedSprite.flip_h = velocity.x < 0
elif velocity.y != 0:
    $AnimatedSprite.animation = "up"
    $AnimatedSprite.flip_v = velocity.y > 0
```

It's looking rather good when playing, doesn't it?

**Comment by chuckdries** [comment]
No, because the walk and up animations look identical. The "walk" animation does not go right or left because the sprite is rotated up

**Comment by chuckdries** [comment]
<img width="486" alt="Screen Shot 2022-07-19 at 5 37 56 PM" src="https://user-images.githubusercontent.com/1351436/179871144-eeb5863f-ed06-4ee8-a425-5226bc1ba774.png">
<img width="509" alt="Screen Shot 2022-07-19 at 5 38 01 PM" src="https://user-images.githubusercontent.com/1351436/179871142-fd9cece9-89c7-47e7-8ac5-d55f887fb662.png">

See in the "walk" animation the sprites should be facing the right

**Comment by ego-edesse** [comment]
Do you mean the little "legs" down the eye, in the "walk" animation, should be rotated +90 degrees clockwise?

**Comment by chuckdries** [comment]
yes. I now realize that the eye is actually pointing a different direction. I just expected the player to work the same way as the mobs and as the player in the 3d version (i.e. it moves the opposite direction of the legs)
